src/clint.py

```diff
@@ -2600,16 +2600,23 @@ def CheckBraces(filename, clean_lines, linenum, error):
         else:
             func_start_linenum = end_linenum + 1
             while not clean_lines.lines[func_start_linenum] == '{':
-                if not Match(r'^(?:\s*\b(?:FUNC_ATTR|REAL_FATTR)_\w+\b(?:\(\d+(, \d+)*\))?)+$',
-                             clean_lines.lines[func_start_linenum]):
+                attrline = Match(r'^(\s*)(?:\b(?:FUNC_ATTR|FUNC_API|REAL_FATTR)_\w+\b(?:\(\d+(, \d+)*\))?\s*)+$',
```

Comment:
Also no need to add `\b`, you are already allowing only spaces or nothing before the regexp. And it does not look like anything is going to break if you do the match like I do with `ag`: only start of the line. Just you would need not “exactly two space indent” diagnostics, rather “previous line indent plus two spaces” (for some REAL_FATTR used in macros).